toqito/channel_metrics/completely_bounded_trace_norm.py

```diff
@@ -9,8 +9,10 @@
 
 def completely_bounded_trace_norm(phi: np.ndarray) -> float:
     r"""
-    Compute the completely bounded trace norm / diamond norm of a quantum
-    channel [WatCNorm18]_. The algorithm in p.11 of [WatSDP09] with
+    Find the completely bounded trace norm of a quantum channel.
+
+    Also known as the completely bounded diamond norm of a quantum
+    channel[WatCNorm18]_. The algorithm in p.11 of [WatSDP09] with
```

Comment:
```suggestion
    channel [WatCNorm18]_. The algorithm in p.11 of [WatSDP09] with
```
might want to add this space here.